WriteHDLUtils.py

```diff
@@ -799,8 +862,9 @@ def writeModuleInst_generic(module, hls_src_dir, f_writeTemplatePars,
                     foundMatch = (argname==portname)
                 else:
                     # Use the provided matching rules
-                    foundMatch = f_matchArgPortNames(argname, portname)
-
+                    if "table" not in argname:
```

Comment:
I guess this could be removed if "if "table" not in argname" is moved outside this else statement (see above comment)